src/configurations/adknowledgeportal/routesConfig.ts

```diff
@@ -35,8 +35,9 @@ import {
 
 const routes: GenericRoute[] = [
   {
-    to: '',
-    isNested: false,
+    path: '',
+    hideRouteFromNavbar: true,
+    exact: true,
```

Comment:
Removed `isNested`, which had custom, undocumented behavior that I'm not sure I could accurately explain. Replaced with `exact`, which comes from [react-router](https://v5.reactrouter.com/web/api/Route/exact-bool), so we get the benefits of react-router documentation.